tools/png2c/palette.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"../misc"
+	"image"
+	"image/color"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type PaletteParams struct {
+	Name        string
+	Colors      int
+	StoreUnused bool
+}
+
+func ParseDoPaletteParams(paletteFlag string) (PaletteParams, error) {
```

Comment:
@kjaroma Wouldn't it be better if you had only single parser for all parameters (i.e. palette, bitmap, etc.) ?

Reply:
@cahirwpz Maybe, I'll think about it, but I'm concerned about the complexity of returning union types from ParseParams function.

Reply:
@cahirwpz I have found the solution for parsing parameters. The flag package provides custom flag functionality.